### PR TITLE
NT-39 Text To Speech

### PR DESCRIPTION
**Pull request by PatrickSlag**

### CODE REVIEW

File: Assets/Scripts/Presenters/SpeechPresenter.cs [hunk 1]
```diff
@@ -7,38 +8,63 @@ namespace Presenters
...
+                else
+                {
+                    _speechToTextService.SpeechTranscribe += OnSpeechToTextTranscribe;   
+                }
+
+                _textToSpeechService = gameObject.AddComponent<WebGLTextToSpeechService>();
+                if (!_speechToTextService.CheckSupport())
+                {
+                    Destroy(GetComponent<WebGLTextToSpeechService>());
+                    Debug.LogWarning("Speech synthesis not supported on your platform or browser");
```
Comment (VarioCode) [inline Assets/Scripts/Presenters/SpeechPresenter.cs:62]: Can we also give feedback to the User inside the UI, not only via a Debug message.

File: Assets/Scripts/Presenters/SpeechPresenter.cs [hunk 2]
```diff
@@ -7,38 +8,63 @@ namespace Presenters
...
         {
             if (Application.platform == RuntimePlatform.WebGLPlayer && !Application.isEditor)
             {
-                _speechService = gameObject.AddComponent<WebGLSpeechService>();
-                if (!_speechService.CheckSupport())
+                _speechToTextService = gameObject.AddComponent<WebGLSpeechToTextService>();
+                if (!_speechToTextService.CheckSupport())
+                {
+                    Destroy(GetComponent<WebGLSpeechToTextService>());
+                    Debug.LogWarning("Speech recognition not supported on your platform or browser");
```
Comment (VarioCode) [inline Assets/Scripts/Presenters/SpeechPresenter.cs:51]: I would like some error message in the UI, to inform the user, no real user is gonna look inside the logging.

**Review by VarioCode — CHANGES_REQUESTED** [review]

**Comment by PatrickSlag** [comment]
https://038games.atlassian.net/browse/NT-246

**Review by VarioCode — APPROVED** [review]

**Review by VarioCode — DISMISSED** [review]

**Review by VarioCode — DISMISSED** [review]

**Review by jortdr — APPROVED** [review]
lgtm

**Review by VarioCode — APPROVED** [review]

**Pull request merged**